Html-Files/news.html

```diff
@@ -632,32 +632,33 @@ <h3>RapiDoc's Research Team Publishes Groundbreaking Study on Preventive Healthc
                 <section>
                     <h4>Quick Links</h4><br>
                       <ul class="links">
-                        <li><a href="./index.html">Home</a></li>
-                        <li><a href="#about">About Us</a></li>
-                        <li><a href="./Html-Files/contact.html">Contact Us</a></li>
-                        <li><a href="#services">Services</a></li>
-                        <li><a href="#doctors">Doctors</a></li>
-                        <li><a href="./faq.html">FAQ</a></li>
+                        <li><a href="/index.html">Home</a></li>
+                        <li><a href="/index.html#about">About Us</a></li>
+                        <li><a href="/Html-Files/contact.html">Contact Us</a></li>
+                        <li><a href="/index.html#services">Services</a></li>
+                        <li><a href="/index.html#doctors">Doctors</a></li>
+                        <li><a href="/faq.html">FAQ</a></li>
                     </ul>
                 </section>
                 <section>
                   <h4>Explore</h4><br>
                   <ul class="links">
-                    <li><a href="#">Appointment</a></li>
-                    <li><a href="#">Patient Portal</a></li>
-                    <li><a href="#">Login account</a></li>
+                    <li><a href="/index.html#appointment">Appointment</a></li>
+                    <li><a href="/index.html#patient-portal">Patient Portal</a></li>
+                    <li><a href="/login.html">Login account</a></li>
                     <li><a href="#">Helplines</a></li>
-                    <li><a href="#">Location</a></li>
+                    <li><a href="/index.html#contact">Location</a></li>
                   </ul>
                 </section>
                 <section>
                   <h4>Legal</h4><br>
                   <ul class="links">
                     <li><a href="#">Customer Agreement</a></li>
-                    <li><a href="#">Privacy Policy</a></li>
+                    <li><a href="/Html-Files/privacy_policy_page.html">Privacy Policy</a></li>
+                    <li><a href="/Html-Files/terms_and_conditions.html">Terms and Conditions</a></li>
                     <li><a href="#">GDPR</a></li>
                     <li><a href="#">Security</a></li>
-                    <li><a href="#">Testimonials</a></li>
+                    <li><a href="/index.html#test">Testimonials</a></li>
```

Comment:
Can be #test? It's now redirecting to top of home page

Reply:
@varshith257  test.html is for blood tests, not testimonials. People were overlooking it and directing to the wrong page. I have made this correction almost everywhere.
A dedicated page for testimonials is yet to be made.